web-client/src/app/transloco/transloco-root.module.ts

```diff
@@ -0,0 +1,38 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable, NgModule } from '@angular/core';
+import {
+  Translation,
+  translocoConfig,
+  TranslocoLoader,
+  TranslocoModule,
+  TRANSLOCO_CONFIG,
+  TRANSLOCO_LOADER,
+} from '@ngneat/transloco';
+import { environment } from '../../environments/environment';
+
+@Injectable({ providedIn: 'root' })
+export class TranslocoHttpLoader implements TranslocoLoader {
+  constructor(private http: HttpClient) {}
+
+  getTranslation(lang: string) {
+    return this.http.get<Translation>(`/assets/i18n/${lang}.json`);
+  }
+}
+
+@NgModule({
+  exports: [TranslocoModule],
+  providers: [
+    {
+      provide: TRANSLOCO_CONFIG,
+      useValue: translocoConfig({
+        availableLangs: ['en', 'fr'],
+        defaultLang: 'en',
```

Comment:
Should we default to the browser language here?

```suggestion
        defaultLang: getBrowserLang(),
        fallbackLang: 'en',
```

Reply:
We still going to need translation files for each language. setting this up right now will just bring unnecessary problems

Reply:
With `fallbackLang` set, it should still be fine, right?

Reply:
Yes, but there is no need to do it if we know for a fact that it will fall back anyway!
We will implement when we have at least 1 additional language fully translated.

(I will do fr after the UI is solid)

Reply:
I think it's worth doing now: it makes it easy to spot-test the localisation by changing the browser's locale. (That's how I tested this PR.)

We'll want to be able to do this kind of quick interactive switching while we develop the rest of the app, to make sure fields are correctly tagged, and to test partial translations; this should be the most low-cost and low-risk way of doing that. (The alternative would be having to change the default in the source code and recompiling every time we want to switch it, but that's a lot worse, I think.)

Since this is something we'll need anyway, and is already useful for development and testing right now, I don't think there's a reason to delay it, is there?

Reply:
>I don't think there's a reason to delay it, is there?

You are totally right, but there is no reason to buy paints if there are no walls yet. 
When we start translating (which will happen @ some point), we will implement it :).






> The alternative would be having to change the default in the source code and recompiling every time we want to switch it

There is no recompiling with this plugin, that's the #1 reason I went with it over the built-in Angular i18n. 
The translation is done JIT

Reply:
No, I mean having to recompile to switch the actual UI for previewing, since we don't have other UI for switching the app's current language dynamically.

Using the browser preference just means we don't have to implement an in-app UI (or change the source default and recompile) to be able to do that.

Reply:
(Resolved in #32.)

Reply:
> * Resolved in: [translate 2 #32](https://github.com/registreerocks/nautilus-wallet/pull/32)

You just had to do it :) ...